#
# Upstream: forwarded by e-mail
# Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=455975
# Description: recognise SUN's LDOM virtual block devices.
#
--- lvm2-2.02.26.orig/lib/filters/filter.c
+++ lvm2-2.02.26/lib/filters/filter.c
@@ -75,6 +75,7 @@
 	{"aoe", 16},		/* ATA over Ethernet */
 	{"device-mapper", 1},	/* Other mapped devices */
 	{"xvd", 16},		/* Xen virtual block device */
+	{"vdisk", 8},		/* SUN's LDOM virtual block device */
 	{NULL, 0}
 };
 
